tools/docker/builder/openwrt/build.sh

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
 scriptdir="$(cd "${0%/*}"; pwd)"
 rootdir="${scriptdir%/*/*/*/*}"
 
+# shellcheck source=../../../../tools/functions.sh
```

Comment:
```suggestion
# shellcheck source=../../../functions.sh
```

Reply:
I disagree - with `../../../../tools/functions.sh` it corresponds one-to-one with the expansion of `$rootdir/tools/functions.sh`